Makefile

```
@@ -0,0 +1,15 @@
+.PHONY: build push help
+.DEFAULT_GOAL := help
+
+IMAGE_NAME := quay.io/3scale/zync
```

```suggestion
IMAGE_NAME ?= quay.io/3scale/zync
```

---

Or maybe even in just one step:

```suggestion
ZYNC_IMAGE := quay.io/3scale/zync:latest
```

---

Why? I do not want to deffer the expansion if any here

---

To be able to control the image name from the outside. To open it up for other usage, like in dev workflows as well. E.g.:

```sh
ZYNC_IMAGE=quay.io/user/zync:my-branch make build
```

(Just a suggestion.)

---

hmmm ... what is the difference, it is a Makefile, you pass the args in the command line in your script:

```shell
make -f zync/Makefile IMAGE_NAME=${ZYNC_IMAGE}
```

---

You're totally right. Apologies.

---

I sometimes leverage exporting variables to the shell once and then running a series of make targets without carrying about the parameters any more. But you don't need that in this case, so never mind.